From abccb52aa01d169fae6e6fc2173ca1d67915ce7b Mon Sep 17 00:00:00 2001
From: Shivani Raval <sraval@habana.ai>
Date: Thu, 05 Jun 2025 08:22:25 +0300
Subject: [PATCH] [JSW-830] build: create images folder for jgs

Create elfs, itbs, hexs folders in jgs/images/
as currently we have to do manually and for all
other projects its done as a part of build.

Change-Id: Id7d357219a047eb2c6768dc89212aa606b905fc2
Signed-off-by: Shivani Raval <sraval@habana.ai>
---

diff --git a/scripts/make/hl-common/Makefile b/scripts/make/hl-common/Makefile
index 055c5df..9e5b9bf 100644
--- a/scripts/make/hl-common/Makefile
+++ b/scripts/make/hl-common/Makefile
@@ -144,6 +144,9 @@
 	@$(if $(filter $(HL_MULTI_ARCH), $@), \
 		$(MAKE) \
 		$(call hl-device, $@)_arc-distclean)
+	@$(if $(filter $(HL_SINGLE_ARCH_RELEASE), $(call hl-device, $@)), \
+		$(MAKE) \
+		hl-common-$(strip $(call hl-device, $@)))
 	@$(MAKE) $@-custom
 
 .PHONY: %-custom
